tempodb/encoding/v2/encoding.go

```diff
@@ -47,7 +47,7 @@ func (v Encoding) OpenWALBlock(filename string, path string, ingestionSlack time
 }
 
 // CreateWALBlock creates a new appendable block
-func (v Encoding) CreateWALBlock(id uuid.UUID, tenantID string, filepath string, e backend.Encoding, dataEncoding string, ingestionSlack time.Duration) (common.WALBlock, error) {
+func (v Encoding) CreateWALBlock(id uuid.UUID, tenantID string, filepath string, e backend.Encoding, dataEncoding string, ingestionSlack time.Duration, dedicatedColumns ...backend.DedicatedColumn) (common.WALBlock, error) {
```

Comment:
I'd prefer to use `dedicatedColumns []backend.DedicatedColumn` instead of a variadic parameter here too